### DISCUSSION TITLE
My github student perk of github pro has been revoked?

### DISCUSSION THREAD

**Discussion by kqnade** [community/community#43285] [GitHub Education]
Forgive my inquiry by translator. I was certified as a Github Student and got Github Pro around September last year. However, I recently noticed that my Pro status was revoked. I thought I needed to re-authenticate, so I went to Github Education and it said I was already authenticated. Also, when I selected my school email address, it said that this email address was already being used for another account and my user ID. I most recently changed my email address and changed my username. Without Github Pro, I can't host Pages privately and my development is stalled. What can I do to get back to Github Pro?

**Comment by jfullstackdev** [comment]
that email is linked to your student package, I think so,

so the big question is, why it's being used, do you have another account on GitHub ? that is a possibility

also, there is no assurance but you can try Support 

https://support.github.com/contact/account

**Reply by kqnade** [reply]
Thanks for the reply.
The github education verification page shows "[my school email] has already been used to obtain an academic discount on GitHub account(s) kqnade."
As you know, kqnade is my username.
I don't have another account.
I also contacted support, but they replied that "any questions about benefits should be directed to the benefits provider" and closed the ticket.
